Justfile

```diff
@@ -266,6 +268,76 @@ gen_test_dir:
 
     # END test_root
 
+    # BEGIN test_symlinks
+
+    mkdir -p symlinks
+    touch symlinks/file --date=@0
+    ln -s file symlinks/symlink
+    ln -s symlink symlinks/symlink2
+    mkdir -p symlinks/dir
+    ln -s dir symlinks/symlink3
+    ln -s pipitek symlinks/symlink4
+
+    # END test_symlinks
+
+    # BEGIN test_perms
+
+    mkdir -p perms
+    touch perms/file --date=@0
+    touch perms/file2 --date=@0
+    chmod 777 perms/file
+    chmod 001 perms/file2
+
+    # END test_perms
+
+    # BEGIN test_group
+    mkdir -p group
+    touch group/file --date=@0
+    sudo chgrp eza_test group/file
+    # END test_group
+
+    # BEGIN test_size
+    mkdir -p size
+    touch size/1M --date=@0
+    dd if=/dev/zero of=size/1M bs=1 count=0 seek=1M
+    touch size/1K --date=@0
+    dd if=/dev/zero of=size/1K bs=1 count=0 seek=1K
+    touch size/1B --date=@0
+    dd if=/dev/zero of=size/1B bs=1 count=0 seek=1
+    touch size/69 --date=@0
+    dd if=/dev/zero of=size/69 bs=1 count=0 seek=69
```

Comment:
\>_>
```suggestion
    dd if=/dev/zero of=size/1337 bs=1 count=0 seek=69
```

Reply:
can't use your suggestion it is not complete but changed the file name and size